scripts/qa_crowbarsetup.sh

```diff
@@ -2893,10 +2899,19 @@ function custom_configuration
                 local maybe_direct_interface=''
                 [[ $deployswift ]] && maybe_direct_interface=", 'direct'"
                 proposal_set_value ironic default "['attributes']['ironic']['enabled_deploy_interfaces']" "['iscsi' $maybe_direct_interface]"
+
+                if [[ $want_tempest ]]; then
```

Comment:
normally we do 

```
[[ $want_tempest = 1 ]]
``` 

in the rest of the code. can you please convert to that? because above will "pass" with want_tempest=0:

```
want_tempest=0 ;  [[ $want_tempest ]] && echo 1
1
```